doc/cla/individual/badbole.md

```diff
@@ -0,0 +1,11 @@
+Croatia, 2015-10-31
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
```

Comment:
no need, if you are signing the corporate CLA, no need to sign the individual one